library/menu/src/PatientMenuRole.php

```diff
@@ -181,15 +181,17 @@ public function displayHorizNavBarMenu()
                 // flatten to only show children items
                 foreach ($value->children as $children_key => $children_value) {
                     $link = ($children_value->pid != "true") ? $children_value->url : $children_value->url . attr($pid);
-                    $list = '<li class="oe-bold-black" id="nav-list'. $li_id.'">';
-                    $list .= '<a href="' . $link . '" onclick="' . $children_value->on_click .'"> ' . text($children_value->label) . ' </a>';
+                    $class = isset($children_value->class) ? $children_value->class : '';
+                    $list = '<li class="oe-bold-black ' . attr($class) . '" id="' . attr($children_value->menu_id) . '">';
+                    $list .= '<a href="' . attr($link) . '" onclick="' . attr($children_value->on_click) .'"> ' . text($children_value->label) . ' </a>';
```

Comment:
oops, lets remove the attr from the $children_value->on_click entry (that can break things)